Example/AppDelegate+CarPlay.swift

```diff
@@ -553,6 +553,9 @@ class CarPlaySceneDelegate: NSObject, CPTemplateApplicationSceneDelegate {
         appDelegate.carPlayManager.templateApplicationScene(templateApplicationScene,
                                                             didConnectCarInterfaceController: interfaceController,
                                                             to: window)
+        if appDelegate.currentAppRootViewController?.activeNavigationViewController != nil {
```

Comment:
Can you please add comment to make sure we remember what's the purpose of this check?